<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Gesture Recognition Framework ImageLab (GRF)</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="boxRGB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
    </property>
    <property name="title">
     <string>RGB Video</string>
    </property>
    <widget class="QLabel" name="labelRGB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>601</width>
       <height>411</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widgetRecognizer" native="true">
     <property name="geometry">
      <rect>
       <x>509</x>
       <y>30</y>
       <width>921</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxDepth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
    </property>
    <property name="title">
     <string>Depth Video and Skeleton</string>
    </property>
    <widget class="QLabel" name="labelDepth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>601</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetR" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>1271</width>
      <height>961</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupSource">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>441</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Sources</string>
     </property>
     <widget class="QRadioButton" name="radioKinect1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioKinect2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioFile">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Images/kinect1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Images/kinect2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Microsoft Kinect 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Microsoft Kinect 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Images/folder-128.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gesture File</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioMyo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Myo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>280</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Images/myo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupMethod">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>421</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Methods</string>
     </property>
     <widget class="QRadioButton" name="radioHMM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hidden Markov Model</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioDTW">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>125</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dinamic Time Warping</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioTest">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>205</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Testing</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioSave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>285</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Save and Annotation Tool</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioTestMyo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Testing Myo Armband</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonStart">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>490</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetL" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1271</width>
       <height>961</height>
      </rect>
     </property>
    </widget>
    <zorder>widgetL</zorder>
    <zorder>groupSource</zorder>
    <zorder>groupMethod</zorder>
    <zorder>buttonStart</zorder>
   </widget>
   <widget class="QLabel" name="labelTimeAcq">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>930</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>930</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeElab">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>930</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>930</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ET:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>930</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>930</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>930</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFPS">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>930</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionClose"/>
   <addaction name="actionExit_2"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

</string>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionProject"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionProject">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionKk">
   <property name="text">
    <string>kk</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
